pkg/tfgen/generate_test.go

```diff
@@ -164,7 +164,8 @@ func Test_makePropertyType(t *testing.T) {
 	}).Shim()
 
 	t.Run("String", func(t *testing.T) {
-		p := g.makePropertyType(path, "obj", strType, nil, false, entityDocs{})
+		p, err := g.makePropertyType(path, "obj", strType, nil, false, entityDocs{})
+		assert.NoError(t, err)
```

Comment:
Nit: If we use an assert here and `makePropertyType` returns an error, it will show an error for L168 and L169. We don't need to show the error for L169, so we should use `require`, which stops the test on error.
```suggestion
		require.NoError(t, err)
```